HelloMixpanel/EventBindingTests/EventBindingTests.m

```diff
@@ -165,7 +173,7 @@ - (void)testUIControlBindings
     XCTAssertEqual([selector selectFromRoot:rootViewController][0], c2, @"c2 should be selected by path");
 
     // Create binding and check state
-    MPUIControlBinding *binding = [MPEventBinding bindngWithJSONObject:eventParams];
+    MPUIControlBinding *binding = [MPEventBinding bindingWithJSONObject:eventParams];
```

Comment:
Are we getting rid of: + (MPEventBinding *)bindngWithJSONObject:(NSDictionary *)object
I noticed it's deprecated and does the same thing as bindingWithJSONObject